integration/airflow/setup.py

```diff
@@ -13,8 +13,7 @@
 __version__ = "1.2.0"
 
 requirements = [
-    "attrs>=20.3",
-    "requests>=2.20.0",
+    "attrs>=22.2.0",
```

Comment:
Airflow 2.1.4 requires `attrs>=20.0,<21.0`, so this should be loosened to include attrs 20.0.x as well.

Reply:
Let me review if this requirement works for all 20.x

Thanks!